src/sidebar/components/filter-status.js

```diff
@@ -134,9 +135,9 @@ function SelectionFilterStatus({ filterState, rootThread }) {
   // Because of the confusion between counts of entities between selected
   // annotations and filtered annotations, don't display the total number
   // when in user-focus mode because the numbers won't appear to make sense.
-  const buttonText = filterState.focusConfigured
-    ? 'Show all'
-    : `Show all (${totalCount})`;
+  // Don't display total count, either, when viewing a direct-linked annotation.
```

Comment:
could this just say?

"... when in user-focus mode [or when viewing a direct-linked annotation] because the numbers won't appear to `make sense."